src/Symfony/Component/HttpFoundation/ResponseHeaderBag.php

```diff
@@ -157,6 +157,14 @@ public function getCacheControlDirective(string $key): bool|string|null
         return $this->computedCacheControl[$key] ?? null;
     }
 
+    /**
+     * Checks if a cache control directive was explicitly set (not auto-computed).
+     */
+    public function hasExplicitCacheControlDirective(string $key): bool
```

Comment:
adding a method is a hint about a new feature
BUT
why adding a method when we could use hasCacheControlDirective or getCacheControlDirective instead?
once this method is removed, we can consider this as a bugfix IMHO
and then, bugfixes should target the lowest affected branch. 6.4 in this case?